bin/main.dart

```diff
@@ -67,11 +67,17 @@ void main(List<String> args) {
 
   // Getting all the dart files for the project
   final dartFiles = files.dartFiles(currentPath, args);
-  if (dependencies.contains('flutter/') &&
-      dartFiles
-          .containsKey('${currentPath}/lib/generated_plugin_registrant.dart')) {
+  final containsFlutter = dependencies.contains('flutter');
```

Comment:
This PR could basically be just changing `flutter/` to `flutter`, but I decided to improve some other parts a bit as well.